rules/windows/collection_email_powershell_exchange_mailbox.toml

```diff
@@ -4,7 +4,7 @@ integration = ["endpoint", "windows"]
 maturity = "production"
 min_stack_comments = "New fields added: required_fields, related_integrations, setup"
 min_stack_version = "8.3.0"
-updated_date = "2023/06/22"
+updated_date = "2023/09/10"
```

Comment:
Do you mean 10th of September, or 9th of October? If the latter, then I'm afraid you will need to do some updating of these updated dates.